schemas/mw-article-delete.js

```diff
@@ -0,0 +1,54 @@
+{
+    "title": "MediaWiki Article Delete",
+    "description": "Represents a MW Article Delete Edit event",
+    "type": "object",
+    "properties": {
+        // global event fields
+        "uri": {
+            "type": "string",
+            "format": "uri",
+            "description": "the unique URI identifying the event"
+        },
+        "request_id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$",
+            "description": "the unique UUID v1 ID of the event derived from the X-Request-Id header"
+        },
+        "event_id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$",
+            "description": "the unique ID of this event; added by the event bus"
+        },
+        "dt": {
```

Comment:
Perhaps call this `event_ts`, to make it clear that it corresponds to `event_id`? It's more important to capture the semantics in the name. The type (unix timestamp vs. datetime) is already captured in the type, and will be obvious to anybody looking at actual data.